game_patch/input/mouse.cpp

```diff
@@ -111,6 +112,70 @@ ConsoleCommand2 ms_cmd{
     "ms <value>",
 };
 
+float scope_sensitivity_factor = 1.0f;
+float scanner_sensitivity_factor = 1.0f;
+
+constexpr float clamp_sensitivity_modifier(float modifier)
+{
+    constexpr const float min = 0.01f; // prevent division by zero in scope sens
+    constexpr const float max = 4.0f;
+    return std::clamp(modifier, min, max);
+}
+
+void patch_scope_and_scanner_sensitivity()
+{
+    AsmWriter{0x004309B1}.fmul<float>(AsmRegMem{&scope_sensitivity_factor});
+    AsmWriter{0x004309DE}.fmul<float>(AsmRegMem{&scanner_sensitivity_factor});
+}
+
+void update_scope_sensitivity()
+{
+    float modifier = clamp_sensitivity_modifier(g_game_config.scope_sensitivity_modifier);
+    scope_sensitivity_factor = rf::scope_sensitivity_constant * (1.0f / modifier);
+}
+
+void update_scanner_sensitivity()
+{
+    float modifier = clamp_sensitivity_modifier(g_game_config.scanner_sensitivity_modifier);
+    scanner_sensitivity_factor = rf::scanner_sensitivity_constant * modifier;
+}
```

Comment:
This suggested change would remove the clamping entirely (aside from when the value is set via command), meaning negative values or values far outside the useful range could be used. It would also result in scope sensitivity being calculated incorrectly.

It would of course allow the player to set the scope factor to 0, which is technically added functionality I suppose when compared to my version... but it's also effectively pointless.

I will not be accepting this suggestion.